ivadomed/loader/adaptative.py

```diff
@@ -17,11 +17,12 @@
 class Dataframe:
     """
     This class aims to create a dataset using an HDF5 file, which can be used by an adapative loader
-    to perform curriculum learning, Active Learning or any other strategy that needs to load samples in a specific way.
+    to perform curriculum learning, Active Learning or any other strategy that needs to load samples
+    in a specific way.
     It works on RAM or on the fly and can be saved for later.
 
     Args:
-        hdf5 (hdf5): hdf5 file containing dataset information
+        hdf5_file (hdf5): hdf5 file containing dataset information
```

Comment:
it’s minor but we’re trying to stick to a convention for variables: https://github.com/neuropoly/spinalcordtoolbox/wiki/Programming%3A-Variables#variables

so in this case: file_hdf5, and same for path_hdf5 below.

which makes me realize: we should add this internal doc for ivadomed coding rules in our wiki (could you open an issue pls?). thanks!

Reply:
Oh cool good to know! And yes I can open an issue.
In this case, `hdf5_file` is actually an open file, so it isn't a `string`, it's a file object. Should I still call it `file_hdf5`? Or should we call it something else?

Reply:
ah! in that case it is _better_ to not call it `file_hdf5` (otherwise that would be super confusing). So yes, I think you can keep your original naming 😊